src/utils.ts

```diff
@@ -5,6 +5,7 @@ import * as path from "path";
 export function castArray<T>(value: T | T[]): T[];
 export function castArray<T>(value: T | readonly T[]): readonly T[];
 export function castArray<T>(value: T | readonly T[]): readonly T[] {
+    // @ts-ignore
```

Comment:
Why was this added now? This function already exists in the current codebase and is fine.

Reply:
Accidentally included, some issue with my local build